static/nodejs_server/query.ts

```diff
@@ -248,13 +285,10 @@ export async function getQueryResult(
   // Get the nl detect-and-fulfill result for the query
   let nlResp = null;
   try {
-    nlResp = await axios
-      // Set "mode=strict" to use heuristic detector, disable using default place,
-      // use a higher SV threshold and avoid multi-verb queries
-      .post(
-        `${apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=${mode}&client=${client}`,
-        {}
-      );
+    nlResp = await axios.post(
+      `${apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=${mode}&client=${client}&detector=${QUERY_DETECTOR}&svThreshold=${svThreshold}`,
```

Comment:
What does client do in NL server?

Reply:
hm, looking through the editors, looks like client was added in this pr: https://github.com/datacommonsorg/website/pull/3785 

tracing through the code though, I'm not sure it's still being used ..

Reply:
oh lol, i added it!  Ah, yes, this goes into the logs.